### ISSUE TITLE
Exploring warmup & learning rate schedulers

### ISSUE THREAD

**Issue by jackapbutler** [OpenBioML/chemnlp#375]
* We currently use a scheduler which has
   * warmup from 0 to max learning rate during first 20%
   * warmdown from max learning rate to 0 during remaining steps
* and have done some [brief experiments into different schedulers](https://www.notion.so/Grid-Search-1B-Wikipedia-10M-optimisation-hyperparameters-dc6efde320cb42b7aed59a5edbc7d3ce)
* but, there's potentially more room to optimise

See this paper for further inspiration https://arxiv.org/abs/2308.04014
